lib/msf/base/serializer/readable_text.rb

```diff
@@ -5,13 +5,65 @@ module Serializer
 # This class formats information in a plain-text format that
 # is meant to be displayed on a console or some other non-GUI
 # medium.
-class ReadableText
+class ReadableText < Rex::Ui::Text::Output::Stdio
 
   #Default number of characters to wrap at.
   DefaultColumnWrap = 70
   #Default number of characters to indent.
   DefaultIndent     = 2
 
+  def self.format_text(txt, fmt)
+    styles = {
+        :bold => '%bld',
+        :cyan =>'%cya',
+        :red => '%red',
+        :green => '%grn',
+        :blue => '%blu',
+        :yellow => '%yel',
+        :white => '%whi',
+        :magenta => '%mag',
+        :black => '%blk',
+        :dark_red => '%dred',
+        :dark_green => '%dgrn',
+        :dark_blue => '%dblu',
+        :dark_yellow => '%dyel',
+        :dark_cyan => '%dcya',
+        :dark_white => '%dwhi',
+        :dark_magenta => '%dmag',
+        :underline => '%und',
+        :plain => '%clr'
+    }
+
+    return "#{styles[fmt]}#{txt}%clr"
+  end
+
+  def self.md_parse(text)
+
+    lines = text.split("\n")
+    lines.map! do |line|
+      if /^\s*#+\s*(.*)$/.match(line) # THIS REGEX LOOKS FOR '#' CHARACTER AND CAPTURES REST OF LINE
+        line = format_text(/^\s*#+\s*(.*)$/.match(line)[1], :bold)
+      elsif /^\s*$/.match(line) # THIS REGEX HELPS US IGNORE A LINE THAT IS COMPLETELY WHITESPACE
+        line = ""
+      elsif /\*+(.[^\*+]+[^\*+])\*+/.match(line) # THIS REGEX HELPS US TO DETECT WORDS THAT ARE SURROUNDED BY ASTERISKS
```

Comment:
I'm assuming you're refrring to line 48 here, since that's the one that has a {{.}} at the beginning of the capture group. You can see what this regex does at the following link:
http://rubular.com/r/SmvANjbZPh